tests/great_international/test_serializers.py

```diff
@@ -1624,26 +1622,25 @@ def test_atlas_opportunity_list_page_related_opportunities(rf, international_roo
         related_sub_sector=subsector_b3
     )
 
-    related_sector_c = InvestmentOpportunityRelatedSectorsFactory(related_sector=sector_c)
-
     # Make eight opportunities:
-    # 1. Region A, Sector A, Subsector A1 -> Thing we'll match against
-    # 2. Region A, Sector A, no subsector -> valid match
-    # 3. Region A, Sector B, Subsector B2 -> valid match
-    # 4. Region A, Sector B, Subsector B3 -> valid match
-    # 5. Region B, Sector A, no subsector -> should not be shown
-    # 6. Region B, Sector A, Subsector A1 -> should not be shown
-    # 7. Region C, no sector -> should not be shown
-    # 8. Region C, Sector C -> should not be shown
-    # 9. Region C and Region A -> valid match
+    # 1. Sector A, Subsector A1 -> Thing we'll match against
+    # 2. Sector A, no subsector -> valid match
+    # 3. Sector B, Subsector B2 -> valid match
+    # 4. Sector B, Subsector B3 -> valid match
+    # 5. Sector A, -> should not be shown
+    # 6. Sector A, Subsector A1 -> should not be shown
+    # 7. Sector C -> should be shown
+    # 8. Sector C -> should  be shown
+    # 9. Sector C -> valid match
+    # 10 with no sector -> should not be shown
```

Comment:
Let's use `related` and `not related` instead of `valid match`/`should not be shown`/`should be shown`

Reply:
Also I don't think the expected results here are correct -- I'd expect to see:

```suggestion
    # 2. Sector A, no subsector -> related
    # 3. Sector B, Subsector B2 -> not related
    # 4. Sector B, Subsector B3 -> not related
    # 5. Sector A, -> related
    # 6. Sector A, Subsector A1 -> related
    # 7. Sector C -> not related
    # 8. Sector C -> not related
    # 9. Sector C -> not related
    # 10 with no sector -> not related
```